grails-app/controllers/org/pih/warehouse/picklist/ImportPickCommand.groovy

```diff
@@ -25,6 +25,27 @@ class ImportPickCommand implements Validateable {
         })
         expirationDate(nullable: true)
         binLocation(nullable: true)
-        quantity(nullable: false, min: 0)
+        quantity(nullable: false, validator: { val, obj ->
+            if (!NumberUtils.isNumber(val)) {
+                return ['notANumber.error']
+            }
+            if (Integer.parseInt(val) < 0) {
+                return ['negative.error']
+            }
+            return true
+        })
+        quantityAsNumber(nullable: true)
```

Comment:
More to the point, is this necessary to resolve some other issue? This feels like it might be related to the fact that the getQuantityAsNumber() getter should probably have a property defined in the transients list.

Reply:
it was necessary because `getQuantityAsNumber()` returned a `null` and without it I got an error that `getQuantityAsNumber` cant be a nul value.
And as Kacper mentioned earlier, we don't have to define a transient for this getter